docs/user-guide/sans-polarization-analysis-methodology.ipynb

```diff
@@ -237,7 +245,8 @@
    "id": "999b0254-3f3f-401b-8b51-424fc4971407",
    "metadata": {},
    "source": [
-    "### Literature: \n",
+    "## Literature\n",
+    "\n",
     "1) Wai Tung Lee et al., Polarisation Development at the European Spallation Source\n",
     "2) Kathryn Krycka et al., Polarization-analyzed small-angle neutron scattering. I. Polarized data reduction using Pol-Corr\n",
     "3) A. Wildes, The polarizer-analyzer correction problem in neutron polarization analysis experiments\n",
```

Comment:
Be consistent with either first names or initials?

Reply:
Unrelated?

Reply:
Again for me to be changed in other PR